content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,382 @@
+
+{{% alert title="Note" %}}
+As noted under [Accusations](/concepts/accountability/#accusations) above, a validator with a _pending accusation_ is not accusable because protocol has to wait to determine if the _pending accusation_ has been defended or, if not, promoted to a fault or not. Until then, protocol cannot determine if the _offending_ validator has committed a rule infraction with a higher _severity_ or not in the epoch.
+{{% /alert %}}
+
+#### Faults 
+
+A _fault_ is a proven consensus rule infraction in an epoch. A _fault_ is created by:
+
+- **direct submission**: a _reporting_ validator submits a _fault proof_ on-chain.  Directly submitted faults:
+  - does not offer possibility of defence
+  - are unforgeable evidence of rule infraction
+
+- **_promotion_ of an _accusation_**:
+  - offers the possibility of defence by submission of an _innocence_ proof
+  - promotion to _fault_ requires the _accusation_ to have a higher _severity_ than any _fault_ the _offending_ validator has for the epoch.
+
+{{% alert title="Note" %}}
+Unlike _accusations_ where an accusation must be made within a `<= 256` block window of the rule infraction, there is no _window_ constraint for direct _fault_ submissions.
+
+A direct fault proof can be reported at any time.
+{{% /alert %}}
+
+Slashing for _faults_ is applied at the end of each epoch. For each validator with _fault(s)_ slashing is applied for the _fault_ with the highest _severity_.
+
+After successful [handling and verification](/reference/api/aut/op-prot/#handleevent-accountability-contract) of a directly submitted _fault_ on-chain a `NewFaultProof` event is emitted logging the _offending_ [validator identifier](/concepts/validator/#validator-identifier) address, _severity_ of rule infraction, and the event ID.
+
+#### Slashing and severity
+
+The AFD protocol will only apply slashing to an _offending_ validator once in an epoch, only slashing the reported _fault_ with the highest _severity_ in an epoch.
+
+Rule infraction _severity_ has two key influences:
+
+- determining if a new _fault_ is created or not, conditional on the _offending_ validator not having an existing _fault_ with a _severity_ `>=` to that of the candidate new _fault_
+- determining the amount of the slashing applied.
+
+Slashing amount is calculated by a number of parameters, including _severity_. 
+
+For the severity taxonomy see [Rule severity](/concepts/accountability/#rule-severity).
+
+For slashing calculation and parameters see [Slashing](/concepts/accountability/#slashing).
+
+{{% alert title="Note" %}}
+A validator can theoretically be slashed for committing a fault in an epoch more than once in an edge case scenario. For example:
+
+- **epoch `n`**: _offending_ validator slashed for highest severity fault committed in epoch `n`
```

Comment:
```suggestion
- **epoch `n`**: _offending_ validator slashed for fault committed in epoch `n`
```

if highest severity that we cannot have a higher severity fault reported in epoch n+1